.github/workflows/get-version.yml

```
@@ -38,12 +34,15 @@ on:
       release_cloud:
         description: "context of release (cloud or not cloud)"
         value: ${{ jobs.get-version.outputs.release_cloud }}
+      pull_request_environment:
+        description: "Pull request environment"
+        value: ${{ jobs.get-version.outputs.pull_request_environment }}
 
 
 jobs:
   get-version:
     runs-on: ubuntu-22.04
-    environment: ${{ inputs.pull_request_environment }}
+    environment: ${{ github.event_name == 'pull_request_target' && github.event.pull_request.head.repo.fork && 'external_contributor' || 'internal_contributor' }}
```

```suggestion
    environment: ${{ (github.event_name == 'pull_request_target' && github.event.pull_request.head.repo.fork) && 'external_contributor' || 'internal_contributor' }}
```
Shouldn't we add parenthesis in the ternary ?